db/migrate/20150222094809_create_taxonomy_terms.rb

```diff
@@ -2,7 +2,7 @@ class CreateTaxonomyTerms < ActiveRecord::Migration
   def change
     create_table :taxonomy_terms do |t|
       t.string :label, null: false, index: true
-      t.string :name, null: false
+      t.string :name, null: false, index: true
```

Comment:
Good point, this should de facto be a unique value. Otherwise there really is no point in having a taxonomy. Will fix it in another issue/PR.